portfolio/src/main/webapp/index.html

```diff
@@ -9,29 +9,43 @@
+      <button onclick="addRandomSong()">Click Me!</button>
+      <div id="song-container"></div>
     </div>
 
-    <h3>Programming Background</h3>
-    <p>The reason why I started programming was so that I did not have to take another History class
-        where all I was going to do was write essays. In my Sophomore year of high school, my school 
-        offered their first Computer Science course that taught by our IT guy. Although I did not
-        like the way he taught the class, I enjoyed the material. After Sophomore year, my school 
-        decided to continue to offer higher and higher CS courses, even if there were only 5 people
-        in the class. From then on, I decided to major in Computer Science at Fisk University, where
-        I started to fall in love with CS.<p>
+    <div id="content">
+        <h3>About Me</h3>
+        <p>My name is Erys Dennis. I am 18 years old and a rising sophomore at Fisk University majoring in 
+            Computer Science. I was born in and currently live in Atlanta, Ga. I have two sisters, one older
+            and one younger. My younger sister is a rising senior in high school. My older sister is currently
+            pursuing her Phd. and lives in Washinton DC. At the present, I have two dogs named Bruno and Moses.
+            Bruno is 1 year-old French Bulldog and Moses is a 12 week-old Bullmastiff.<p>
+
+        <h3>Programming Background</h3>
+        <p>The reason why I started programming was so that I did not have to take another History class
+            where all I was going to do was write essays. In my Sophomore year of high school, my school 
+            offered their first Computer Science course that taught by our IT guy. Although I did not
+            like the way he taught the class, I enjoyed the material. After Sophomore year, my school 
+            decided to continue to offer higher and higher CS courses, even if there were only 5 people
+            in the class. From then on, I decided to major in Computer Science at Fisk University, where
+            I started to fall in love with CS.<p>
+
+        <h3>Here is a link to my GitHub!</h3>
+        <a href="https://github.com/erysjd" target="_blank">My GitHub</a> <!-- add CSS styling? -->
+    </div>
+
+    <div id="content">
+      <h3>More Pics of my Dogs!</h3>
+        <a href="images/bruno-sitting.jpg"><img src="images/bruno-sitting.jpg"/></a>
+        <a href="images/bruno-sleep.JPG"><img src="images/bruno-sleep.JPG"/></a>
+        <a href="images/cute-sleep.JPG"><img src="images/cute-sleep.JPG"/></a>
+        <a href="images/funny-face.jpg"><img src="images/funny-face.jpg"/></a>
+        <a href="images/moses-sleep.JPG"><img src="images/moses-sleep.JPG"/></a>
+        <a href="images/moses.jpg"><img src="images/moses.jpg"/></a>
+        <a href="images/playing.JPG"><img src="images/playing.JPG"/></a>
+        <a href="images/week8.JPG"><img src="images/week8.JPG"/></a>
```

Comment:
It doesn't look like you added these images, did you forget to "git add"?

Reply:
I did add them, they have been showing up when I preview it on cloud shell. I just added them again to make sure. Have they not been showing up?

Reply:
Yea, you need to add them to git in addition to adding them to the cloud shell.  "git add" + "git commit" should do that.  I don't see them here yet though

Reply:
Ok. I just added and committed them.